src/main/java/world/bentobox/bentobox/api/commands/island/IslandBanCommand.java

```diff
@@ -74,7 +74,7 @@ public boolean execute(User user, String label, List<String> args) {
         }
         User target = User.getInstance(targetUUID);
         // Cannot ban ops
-        if (target.isOp()) {
+        if (target.isOp() || target.hasPermission("admin.noban")) {
```

Comment:
Use that instead, as `User#hasPermission(...)` returns true if the User is op.
```suggestion
        if (target.hasPermission("admin.noban")) {
```